docs/docs/markdown.md

```diff
@@ -28,6 +28,25 @@ await Docset.Build("docfx.json", options);
 
 Here is a list of markdown extensions provided by docfx by default.
 
+## YAML header
+
+The YAML header is used to annotate a Markdown file with a unique identifier for that document.  It should appear at the top of the document.  Here's an example:
```

Comment:
The YAML header can contain more than just UIDs, it can contain any metadata key value pairs targeted at file level: https://dotnet.github.io/docfx/docs/config.html#metadata 

Here is a list of predefined metadata:
https://dotnet.github.io/docfx/docs/config.html#predefined-metadata

Reply:
Noted.  Added comments to that effect and link to the list of predefined metadata.  Included in next commit.